src/logging.rs

```diff
@@ -0,0 +1,69 @@
+//! Logging related utilities.
+
+use log::{self, Log, Record, Level, Metadata, SetLoggerError};
+
+use super::Sentry;
```

Comment:
I'm not as big of a fan of `super` for something that's in the same directory. I know this is a nitpick but would you mind changing this to: `use Sentry;`?

Reply:
No problem, will update this.